language-specification/lexic.txt

```diff
@@ -0,0 +1,31 @@
+Alphabet:
+
+a. Upper (A-Z) and lower case letters (a-z) of the English alphabet
+b. Underline character '_';
+c. Decimal digits (0-9);
+
+Lexic:
+
+a. Special symbols, representing:
+- operators + - * / % = < <= == > >= != !, and, or
```

Comment:
You also use the ++ operator in P3, which isn't defined here

Reply:
Done!